.bazelrc

```diff
@@ -21,3 +21,10 @@ build --features=layering_check
 # Set the default compiler to the `clang` binary on the `PATH`.
 # TODO(fzakaria): Make this a toolchain or hermetic somehow
 build --repo_env=CC=clang
+
+# Disabling runfiles links drastically increases performance in slow disk IO situations
+# Do not build runfile trees by default. If an execution strategy relies on runfile
+# symlink teee, the tree is created on-demand. See: https://github.com/bazelbuild/bazel/issues/6627
+# and https://github.com/bazelbuild/bazel/commit/03246077f948f2790a83520e7dccc2625650e6df
+build --nobuild_runfile_links
+test --nobuild_runfile_links
```

Comment:
Nit: Looks like this file is missing a newline at EOF. The whitespace checks script only checks a subset of files, not sure why that is: https://github.com/openxla/stablehlo/blob/main/build_tools/github_actions/lint_whitespace_checks.sh#L43

Consider adding a newline.